src/Schema/Type/ShortcodeMarkdown.php

```diff
@@ -32,15 +32,12 @@ public function shortcodeHandler(string $data)
 
         $type = $attributes['type'] ?? '';
 
-        switch ($type) {
-            case 'youtube':
-                return $this->embeddedYoutube($attributes);
-            case 'google-maps':
-                return $this->embeddedGoogle($attributes);
-            case 'thinglink':
-                return $this->embeddedThingLink($attributes);
-        }
-        return null;
+        return match ($type) {
```

Comment:
Using PHP 8 syntax like this will break backwards compatibility. Is this intentional?

Reply:
There are also a bunch of types everywhere in functions that will break compatibility and the minimum PHP version is defined in the composer file.